nixos/modules/hardware/video/nvidia.nix

```diff
@@ -52,6 +52,15 @@ in
     ];
 
   options = {
+    hardware.nvidia.powerManagement.enable = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        Experimental power management through systemd. For more information, see
+        the NVIDIA docs, on Chapter 21. Configuring Power Management Support.
```

Comment:
If these docs are publicly available, then a link would be nice.

Reply:
The docs are within each package, so pinning the doc has the possibility of the docs becoming outdated (I'm not sure if there's a link for the "latest" package).

Then again, I could probably do something along the lines of `https://download.nvidia.com/XFree86/Linux-x86_64/${nvidia_x11.version}/README/powermanagement.html` but not sure if a package evaluation should be done just for the option.

Reply:
Fair :)